components/Wallet/EditWalletForm.tsx

```diff
@@ -27,7 +28,13 @@ export default function EditWalletForm ({ wallet, userPaybuttons, refreshWalletL
     if (params.name === '' || params.name === undefined) {
       params.name = wallet.name
     }
-    console.log('edit wallet params', params)
+    void await fetch(`/api/wallet/${wallet.id}`, {
+      method: 'PATCH',
```

Comment:
Optionally: `axios` would be more succinct. We're using it in `pages/api/resolvers.index.ts`, at least.
Something like:

```js
void await axios.patch(`${websiteDomain}/api/wallet/${wallet.id}`, params );
```

Should accomplish the same thing I think.

Reply:
Addressed (check last commit)